components/ws-manager/pkg/manager/annotations_test.go

```diff
@@ -28,6 +28,10 @@ func TestMarkWorkspace(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.Description, func(t *testing.T) {
 			manager := forTestingOnlyGetManager(t)
+			if manager == nil {
```

Comment:
I think `forTestingOnlyGetManager` have to return the `err` and check `err != nil`. WDYT?

Reply:
It does not return `err` though, and I did not want to change a whole lot. So that seemed like a fairly viable fix, especially if manager failed to get created, you will get an actual error in the test, just no panic anymore, which was very confusing.

Reply:
:ok: I'll fix it again when I get a chance.